charts/redis-ha/templates/redis-ha-configmap-init-s3.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "redis-ha.fullname" . }}-init-s3-env-cm
+  namespace: {{ .Release.Namespace }}
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "redis-ha.fullname" . }}
+data:
+  AWS_ACCESS_KEY_ID: "{{ .Values.restore.s3.access_key }}"
```

Comment:
Any reason why we don't just store this in the secret as well? Saves having a template.